src/transformers/models/idefics/configuration_idefics.py

```diff
@@ -195,9 +215,9 @@ def check_compatibilities(self):
     @classmethod
     def from_pretrained(cls, pretrained_model_name_or_path: Union[str, os.PathLike], **kwargs) -> "PretrainedConfig":
         outputs = super(IdeficsConfig, cls).from_pretrained(pretrained_model_name_or_path, **kwargs)
-        if isinstance(outputs, Tuple):
-            # When called with return_unused_kwargs=True, the first item will be the config
-            outputs[0].check_compatibilities()
-        else:
-            outputs.check_compatibilities()
+        # if isinstance(outputs, Tuple):
```

Comment:
```suggestion
        # XXX: solve this
        # if isinstance(outputs, Tuple):
```
let's flag it to be solved